gnd/src/main/java/com/google/android/gnd/repository/local/RecordEntity.java

```diff
@@ -37,20 +37,33 @@
   @NonNull
   public abstract String getId();
 
+  /**
+   * Returns the id of the form to which this record's responses apply.
```

Comment:
Should these changes be in this PR?

Reply:
Nope, these should go away when I merge PR #19.